nototools/shape_diff.py

```diff
@@ -124,6 +126,16 @@ def find_rendered_diffs(self, font_size=256, stats=None):
             subprocess.call([
                 'convert', '-gravity', 'center', '-background', 'black',
                 '-extent', img_info[2], b_png, b_png])
+
+            if render_path:
+                output_png = os.path.join(render_path, name + '.png')
+                subprocess.call([
```

Comment:
It might be useful here to document what you're doing by specifying each of these options.